src/DsaLocal.cc

```diff
@@ -735,10 +735,7 @@ void IntraBlockBuilder::visitCallSite(CallSite CS) {
   LOG("dsa-callsite", errs()<<"Inst:"<<inst->getName()<<"\n");
   //TODO: not clear if we should create a new node or find an existing one in local graph
 
-  sea_dsa::DsaCallSite *callsite = m_graph.mkDsaCallSite(*inst);
```

Comment:
`mkDsaCallSite` should check that call site already exists and return the existing one.
We should not add it twice, but at least it will not be an issue